trino/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -2310,7 +2310,7 @@ public NotificationEventResponse getNextNotification(
       return rsp;
     } else {
       NotificationEventResponse filtered = new NotificationEventResponse();
-      if (rsp != null && rsp.getEvents() != null) {
+      if (rsp.getEvents() != null) {
```

Comment:
Since `HiveMetaStoreClient` is a class copied from hive 2.11, we may not need to fix these warnings unless they have a substantial impact on correctness.